File: java-network/src/main/java/model/errors/ErrorResponse.java
package model.errors;

import com.google.gson.JsonObject;

/**
 * Created by jc on 5/31/18.
 */
public class ErrorResponse {

    int code;
    String message;
    JsonObject error;

    public int getCode() {
        return code;
    }

    public String getMessage() {
        return message;
    }

    public JsonObject getError() {
        return error;
    }
}

File: java-network/src/main/java/com/eosio/model/errors/ErrorResponse.kt
package com.eosio.model.errors

import com.google.gson.JsonObject

/**
 * Created by jc on 5/31/18.
 */
class ErrorResponse {

    var code: Int = 0
        internal set
    var message: String? = null
        internal set
    var error: JsonObject? = null
        internal set
}
